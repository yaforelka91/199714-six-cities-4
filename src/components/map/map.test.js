import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

const offers = [
  {
    id: 1,
    coords: [52.3909553943508, 4.85309666406198],
  }
];

describe(`MapSnapshot`, () => {
  it(`should render Map`, () => {

    const tree = renderer.create(
        <Map
          offers={offers}
          city={[52, 4.8]}
          zoom={10}
        />,
        {
          createNodeMock: () => document.createElement(`div`)

        }
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
