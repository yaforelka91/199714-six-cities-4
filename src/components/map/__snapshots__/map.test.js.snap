// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapSnapshot should render Map 1`] = `
<section
  className=" map"
>
  <div
    style={
      Object {
        "height": "100%",
      }
    }
  />
</section>
`;
